recipes/libtool/all/conanfile.py

```diff
@@ -220,22 +202,36 @@ def package_info(self):
             if self.settings.os == "Linux":
                 self.cpp_info.system_libs = ["dl"]
 
+        # Use ACLOCAL_PATH to access the .m4 files provided with libtool
+        aclocal_path = os.path.join(self.package_folder, "res", "aclocal")
+        self.buildenv_info.append_path("ACLOCAL_PATH", aclocal_path)
+
+        # Everything below can be eliminated when Conan 1.x support is dropped
         bin_path = os.path.join(self.package_folder, "bin")
-        self.output.info("Appending PATH env: {}".format(bin_path))
+        self.output.info(f'Appending PATH env: {bin_path}')
         self.env_info.PATH.append(bin_path)
 
         bin_ext = ".exe" if self.settings.os == "Windows" else ""
 
-        libtoolize = tools.unix_path(os.path.join(self.package_folder, "bin", "libtoolize" + bin_ext))
-        self.output.info("Setting LIBTOOLIZE env to {}".format(libtoolize))
+        libtoolize = os.path.join(self.package_folder, "bin", "libtoolize" + bin_ext)
+        libtoolize = "/" + libtoolize.replace("\\", "/").replace(":", "") # Can't use unix_path with Conan 2.0
+        self.output.info(f'Setting LIBTOOLIZE env to {libtoolize}')
         self.env_info.LIBTOOLIZE = libtoolize
 
-        for key, value in self._libtool_relocatable_env.items():
-            self.output.info("Setting {} environment variable to {}".format(key, value))
+        libtool_relocatable_env = {
+            "LIBTOOL_PREFIX": self.package_folder,
```

Comment:
If the package is relocatable as per the patch, these are not needed anymore - these variables, confusingly, were a construct of the recipe and not part of the upstream code.

Reply:
Hi @jcar87,
Good point.  I have removed them per your suggestion,. Thanks